### ISSUE TITLE
[Enhancement] Server mode supports PostgreSQL database

### ISSUE THREAD

**Issue by magestacks** [opengoofy/hippo4j#969]

**Comment by Icazio** [comment]
I am interested in this project. Could you please assign it to me? I will send my proposal to ospp for this project.

**Comment by magestacks** [comment]
This Issue is currently participating in the Open Source Summer, and it can only be claimed if no students claim it after 6.3

**Comment by Icazio** [comment]
Yes, I am a student and I want to claim it! I will send my proposal in the Open Source Summer for this project

**Comment by magestacks** [comment]
According to the process, you need to submit the application first, and the Hippo4j community will assign tasks after screening.

**Comment by Icazio** [comment]
np, will submit it asap
